cluster-sim/cluster_sim/fake_server.py

```diff
@@ -91,6 +91,7 @@ class FakeServer(utils.Persisted):
         'lnet_up': False,
         'stats': None,
         'packages': {},
+        'client_mounts': {},
```

Comment:
this was required because when resetting state, at the beginning of one of the tests this key was expected